docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
I think either way we should go for `null` here, if we decide `undefined` is the right choice, then another PR should be opened changing ALL `null`s to `undefined` to ensure consistency. Inconsistent coding style seriously harms readability and maintainability since all new contributors will need to learn too many different semantically identical things.